lvmd/embed.go

```diff
@@ -29,6 +31,20 @@ func NewEmbeddedServiceClients(ctx context.Context, dcmapper *DeviceClassManager
 			log.FromContext(ctx).Error(err, "embedded channel watch error")
 		}
 	}()
+
+	go func() {
+		ticker := time.NewTicker(10 * time.Minute)
```

Comment:
Thanks for the fix idea! QQ: How does the regular trigger for notify work on non-embedded mode? Im wondering how its able to do this refresh in the non-embedded mode

Reply:
NVM, just found https://github.com/topolvm/topolvm/blob/baaea34e4eedbda153232ad01f62504ddcf88395/pkg/lvmd/cmd/root.go#L105-L117

Should we extract this into a common method and also make the time interval modifiable now?

Reply:
Yes, that's a great idea. But I would like to leave that as a separate task, as my priority is to merge this fix and include it in next month's release.